2025/selection-process.md

```diff
@@ -60,7 +60,7 @@ and weigh the shared criteria as they choose.
 
 ### Proposal Submission
 
-**Timeline**: September 12th to October 3rd
+**Timeline**: September 17th to October 7th
```

Comment:
Why don't we make this October 10th (or 9th if it should unambiguously be before the meeting)?

Reply:
Oct 9th makes sense to me